Mathlib/LinearAlgebra/Finsupp/Defs.lean

```diff
@@ -164,6 +164,18 @@ def mapDomain.linearEquiv (f : α ≃ α') : (α →₀ M) ≃ₗ[R] (α' →₀
   right_inv _ := by
     simp [← mapDomain_comp]
 
+@[simp] theorem mapDomain.linearEquiv_coe (f : α ≃ α') :
+    (linearEquiv M R f : _ → _) = mapDomain f := rfl
+
+@[simp] theorem mapDomain.linearEquiv_toLinearMap (f : α ≃ α') :
+    (linearEquiv M R f : _ →ₗ[R] _) = lmapDomain M R f := rfl
+
+@[simp] theorem mapDomain.linearEquiv_symm (f : α ≃ α') :
+    (linearEquiv M R f).symm = linearEquiv M R f.symm := rfl
```

Comment:
I worry all these names are backwards; can you point to existing lemmas they immitate?

Reply:
I guess the `coe` one is the one that is most obviously backwards to me, I'm ok leaving the other two.

Reply:
* ["_coe", ⊢ Subtype.val _ = _](https://loogle.lean-lang.org/?q="_coe"%2C+⊢+Subtype.val+_+%3D+_): 450 hits
* ["coe_", ⊢ Subtype.val _ = _](https://loogle.lean-lang.org/?q="coe_"%2C+⊢+Subtype.val+_+%3D+_): 537 hits

Reply:
* ["coe_", ⊢ DFunLike.coe _ = _](https://loogle.lean-lang.org/?q=%22coe_%22%2C+%E2%8A%A2+DFunLike.coe+_+%3D+_): 1860 hits
* ["_coe", ⊢ DFunLike.coe _ = _](https://loogle.lean-lang.org/?q=%22_coe%22%2C+%E2%8A%A2+DFunLike.coe+_+%3D+_): 192 hits

Reply:
changed all of them

Reply:
I think changing all of them means we get worse for the `symm` case
- [`Equiv.symm _ = _, "_symm"`](https://loogle.lean-lang.org/?q=Equiv.symm+_+%3D+_%2C+%22_symm%22), 108 hits
- [`Equiv.symm _ = _, "symm_"`](https://loogle.lean-lang.org/?q=Equiv.symm+_+%3D+_%2C+%22symm_%22), 41 hits, but in fact 31 of these are actually the previous pattern with `symm_eq`: [`Equiv.symm _ = _, "symm_eq"`](https://loogle.lean-lang.org/?q=Equiv.symm+_+%3D+_%2C+%22symm_eq%22), so this is really 10 hits.

And for the linear version:
- [`LinearEquiv.symm _ = _, "_symm"`](https://loogle.lean-lang.org/?q=LinearEquiv.symm+_+%3D+_%2C+%22_symm%22), 52 hits
- [`LinearEquiv.symm _ = _, "symm_"`](https://loogle.lean-lang.org/?q=LinearEquiv.symm+_+%3D+_%2C+%22symm_%22), 6 hits

Reply:
Changed `symm` back.

Reply:
Happy @b-mehta?

Reply:
Yup!